unskript-ctl/unskript_ctl.py

```diff
@@ -81,6 +81,11 @@ def load_or_create_global_configuration():
         if config_yaml.get('checks'):
             if config_yaml.get('checks').get('arguments'):
                 if config_yaml.get('checks').get('arguments').get('global'):
+                    # Handle matrix
```

Comment:
My be put in a try catch?  because if the arguments section is commented out by user, this code will result in raising exception.

Reply:
why, it has a nested check for each key, no?